### DISCUSSION TITLE
My files don't seem to compress a whole lot. Maybe I'm missing something?

### DISCUSSION THREAD

**Discussion by biggy726** [aonez/Keka#1431] [General]
Hello. So I purchased Keka from the App Store yesterday. The reviews are what sold me on it. However, after trying a few .zip compressions (at slowest speed)  I've had 1 file out of several to see a significant reduction in size. I am attaching a screenshot to give an idea of what I am talking about. Again, this is probably something I am not doing, but I could use some help.

![Screenshot 2024-04-16 at 9 13 21 AM](https://github.com/aonez/Keka/assets/19307813/11e09239-4efb-4b3b-9a41-ca0a89c6374d)

**Comment by gingerbeardman** [comment]
This is all to be expected. 

Why? The amount a file can be compressed depends on the contents.

Imagine a text file:
- AAAAAAAAAAAAAAAAAAAAAAAAA 
- this can be described as "26 x A"

But
- ABCDEFGHIJKLMNOQRSTUVWXYZ
- this cannot be described any more succinctly

So some file formats you're trying to compress are already compressed, to varying degrees, so they are more like ABC... than AAA...

In your examples JPG, PDF, PPTX, Pages (at one time pages files were actually zip files!). You'll learn this as you spend more time dealing with files. 

Some of the apps used to create these have a means to reduce their file size. In Pages and Powerpoint they have menu items to control the compression, whether to delete unused items, etc. JPG has various methods of reducing the size.

What are your goals with these zips? For distrubution (the original purpose of zip was to combine multiple files into one and compress them all so that the single zip was at least smaller than the sum of the totals). Backups? Reducing file space on your computer? Or some other goal?

**Reply by biggy726** [reply]
Thanks so much. Also, I apologize for asking something that should have been obvious. Thank you for taking the time to reply.

Have A Great Day,
Robert
